core/vm/actor/builtin/types/miner/monies.hpp

```diff
@@ -0,0 +1,139 @@
+/**
+ * Copyright Soramitsu Co., Ltd. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#pragma once
+
+#include "vm/actor/builtin/states/miner/miner_actor_state.hpp"
+#include "common/outcome.hpp"
+#include "common/smoothing/alpha_beta_filter.hpp"
+#include "const.hpp"
+#include "primitives/types.hpp"
+#include "vm/actor/builtin/types/miner/policy.hpp"
+#include "vm/runtime/runtime.hpp"
+#include "vm/version/version.hpp"
+
+namespace fc::vm::actor::builtin::types::miner {
+  using common::smoothing::FilterEstimate;
+  using primitives::BigInt;
+  using primitives::ChainEpoch;
+  using primitives::StoragePower;
+  using primitives::TokenAmount;
+  using miner::VestSpec;
+  using states::MinerActorState;
+  using version::NetworkVersion;
+
+  class Monies {
+   public:
+    const int precommit_deposit_factor = 20;
```

Comment:
Please check my [comment](https://github.com/filecoin-project/cpp-filecoin/pull/445#discussion_r694593942).
Use constexpr in all your code.

Reply:
fixed